Formula/cvc5.rb

```diff
@@ -25,18 +24,17 @@ class Cvc5 < Formula
   end
 
   def install
-    venv = virtualenv_create(libexec)
+    venv = virtualenv_create(buildpath/"venv", "python3")
     venv.pip_install resources
 
     command_line = [
       "./configure.sh",
       "--auto-download",
       "--static",
       "--prefix=#{prefix}",
-      "-DCMAKE_FIND_FRAMEWORK=NEVER",
+      "-DPython_EXECUTABLE=#{buildpath}/venv/bin/python",
```

Comment:
do we actually need this line? configure.sh should figure this out, no?

Reply:
Yes, it is actually necessary. Otherwise `configure.sh` selects the Python binary available globally instead of the one in the virtual environment.

Reply:
does this mean that the Python environment is not activated at this stage?

Reply:
Hmm yeah that is weird, you'd think that the environment is properly activated so that python actually links to the correct version in the environment. Alright let's keep it.